plugins/lua/orders.lua

```diff
@@ -107,6 +119,27 @@ function OrdersOverlay:init()
                 auto_width=true,
                 on_activate=do_clear,
             },
+            widgets.HotkeyLabel{
+                frame={t=0, l=30},
+                label='sort by freq',
+                key='CUSTOM_CTRL_O',
+                auto_width=true,
+                on_activate=do_sort,
+            },
+            widgets.HotkeyLabel{
+                frame={t=1, l=30},
+                label='sort by type',
+                key='CUSTOM_CTRL_T',
+                auto_width=true,
+                on_activate=do_sort_type,
+            },
+            widgets.HotkeyLabel{
+                frame={t=0, l=52},
+                label='sort by mat',
+                key='CUSTOM_CTRL_T',
```

Comment:
(not a maintainer, just was curious and looking)

Looks like this hot tea label accidentally has the same key to activate as the one for type!

Reply:
Thanks! It should be M.